src/release/breaking-changes/material-3-migration.md

```diff
@@ -0,0 +1,515 @@
+    const NavigationDrawerDestination(
+      icon: Icon(Icons.settings),
+      label: Text('Settings'),
+    ),
+  ],
+),
+```
+
+Check out the complete sample on [migrating from Drawer from NavigationDrawer][].
+
+Material 3 introduces medium and large app bars that display a larger headline
+before scrolling. Instead of a drop shadow, `ColorScheme.surfaceTint` color
+is used create a separation from the content when scrolling.
+
+The following code demonstrates how to implement the medium app bar:
+
+```dart
+CustomScrollView(
+  slivers: <Widget>[
+    const SliverAppBar.medium(
+      title: Text('Title'),
+    ),
+    SliverToBoxAdapter(
+      child: Card(
+        child: SizedBox(
+          height: 1200,
+          child: Padding(
+            padding: const EdgeInsets.fromLTRB(8, 100, 8, 100),
+            child: Text(
+              'Here be scrolling content...',
+              style: Theme.of(context).textTheme.headlineSmall,
+            ),
+          ),
+        ),
+      ),
+    ),
+  ],
+),
+```
+
+There are now two types of [`TabBar`][] widgets: primary and secondary.
+Secondary tabs are used within a content area to further separate
+related content and establish hierarchy. Check out the [`TabBar.secondary`][]
+example.
+
+The new [`TabBar.tabAlignment`][] property specifies the horizontal alignment
+of the tabs.
+
+The following sample shows how to modify tab alignment in a scrollable `TabBar`:
+
+```dart
+AppBar(
+  title: const Text('Title'),
+  bottom: const TabBar(
+    tabAlignment: TabAlignment.start,
+    isScrollable: true,
+    tabs: <Widget>[
+      Tab(
+        icon: Icon(Icons.cloud_outlined),
+      ),
+      Tab(
+        icon: Icon(Icons.beach_access_sharp),
+      ),
+      Tab(
+        icon: Icon(Icons.brightness_5_sharp),
+      ),
+    ],
+  ),
+),
+```
+
+[`SegmentedButton`][], an updated version of [`ToggleButtons`][],
+uses fully rounded corners, differs in layout height and
+size, and uses a Dart `Set` to determine selected items.
+
+Code before migration:
+
+```dart
+enum Weather { cloudy, rainy, sunny }
+
+ToggleButtons(
+  isSelected: const [false, true, false],
+  onPressed: (int newSelection) { },
+  children: const <Widget>[
+    Icon(Icons.cloud_outlined),
+    Icon(Icons.beach_access_sharp),
+    Icon(Icons.brightness_5_sharp),
+  ],
+),
+```
+
+Code after migration:
+
+```dart
+enum Weather { cloudy, rainy, sunny }
+
+SegmentedButton<Weather>(
+  selected: const <Weather>{Weather.rainy},
+  onSelectionChanged: (Set<Weather> newSelection) { },
+  segments: const <ButtonSegment<Weather>>[
+    ButtonSegment(
+      icon: Icon(Icons.cloud_outlined),
+      value: Weather.cloudy,
+    ),
+    ButtonSegment(
+      icon: Icon(Icons.beach_access_sharp),
+      value: Weather.rainy,
+    ),
+    ButtonSegment(
+      icon: Icon(Icons.brightness_5_sharp),
+      value: Weather.sunny,
+    ),
+  ],
+),
+```
+
+Check out the complete sample on
+[migrating from ToggleButtons from SegmentedButton][].
```

Comment:
There are two "froms" here. Consider something like:

```
For more guidance, check out another sample of
[migrating from `ToggleButtons` to `SegmentedButton`][].
```

(Do upgrade the below link definition as well to whatever you decide :)